server/utils/lostBedUtils.ts

```diff
@@ -0,0 +1,22 @@
+import type { TemporaryAccommodationLostBed as LostBed } from '@approved-premises/api'
+
+export const allStatuses: Array<{ name: string; id: LostBed['status']; tagClass: string }> = [
+  {
+    name: 'Active',
+    id: 'active',
+    tagClass: 'govuk-tag--blue',
+  },
+  {
+    name: 'Cancelled',
+    id: 'cancelled',
+    tagClass: 'govuk-tag--red',
+  },
+]
+
+export function statusTag(statusId: LostBed['status'], viewBedspacePage?: boolean) {
```

Comment:
Maybe change `viewBedspacePage: boolean` to something like `context: 'bookingsAndVoid' | 'voidsOnly'`, so this util doesn't have specific knowledge of a particular page, and give some description as to why there's this variation